docs: update readme with cron and cronjob information (#6)
eduNEXT/platform-plugin-forum-email-notifier: README.rst

Reviewer: We should use the `security@edunext.co` email in the reporting of security issues.

Author: fixed

Forum Email Notifier
####################################

> |pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|

Reviewer: We should only leave the labels that apply

Deploying
=========

> The email digest feature works by acumulating the notifications in a database table
> per user, per course and per digest frequency. Then, a scheduled task is run to
> send the notifications to the users.

Reviewer 2: ```suggestion
The email digest feature accumulates the notifications in a database table
per user, per course, and digest frequency. Then, a scheduled task is run to
send the notifications to the users.
```

> As Open edX doesn't support celery beat for scheduled tasks, so we need to use

Reviewer 2: ```suggestion
As Open edX doesn't support celery beat for scheduled tasks, we need to use
```

another tool to run them.

> For tutor local installations we need to use cron to run the scheduled tasks.

Reviewer 2: cron? can we add a link to the documentation?

Reporting Security Issues
*************************

Please do not report security issues in public. Please email security@edunext.co.

.. It's not required by our contractor at the moment but can be published later
.. .. |pypi-badge| image:: https://img.shields.io/pypi/v/platform_plugin_forum_email_notifier.svg
    :target: https://pypi.python.org/pypi/platform_plugin_forum_email_notifier/
    :alt: PyPI

.. |ci-badge| image:: https://github.com/eduNEXT/platform_plugin_forum_email_notifier/workflows/Python%20CI/badge.svg?branch=main

>     :target: https://github.com/eduNEXT/platform_plugin_forum_email_notifier/actions

Reviewer: ```suggestion
    :target: https://github.com/eduNEXT/platform-plugin-forum-email-notifier/actions
```

    :alt: CI

.. |license-badge| image:: https://img.shields.io/github/license/eduNEXT/platform_plugin_forum_email_notifier.svg

>     :target: https://github.com/eduNEXT/platform_plugin_forum_email_notifier/blob/main/LICENSE.txt

Reviewer: ```suggestion
    :target: https://github.com/eduNEXT/platform-plugin-forum-email-notifier/blob/main/LICENSE.txt
```
